userspace/libscap/engine/gvisor/gvisor.h

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#pragma once
+
+#include <stdint.h>
+#include <string>
+#include <thread> 
+#include <atomic>
+#include <deque>
+#include <vector>
+
+#include "scap.h"
+
+#define GVISOR_MAX_READY_SANDBOXES 32
+#define GVISOR_MAX_MESSAGE_SIZE 300 * 1024
+#define GVISOR_INITIAL_EVENT_BUFFER_SIZE 32
+
+namespace scap_gvisor {
+
+#pragma pack(push, 1)
+struct header
+{
+	uint16_t header_size;
+	uint32_t dropped_count;
+};
+#pragma pack(pop)
+
+struct parse_result {
```

Comment:
Mmmh `parse_result` sounds part of `parsers` namespace; wdyt?

Reply:
Makes sense! Thanks!